src/ClientNet462/Program.cs

```diff
@@ -0,0 +1,116 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Client
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            MainAsync(default).Wait();
+        }
+
+        static async Task MainAsync(CancellationToken cancellationToken)
+        {
+            Console.WriteLine("Searching for city 'detroit'...");
+            var result = await CallGraphQlAsync<MyResponseData>(
+                new Uri("https://graphql-weather-api.herokuapp.com/"),
+                HttpMethod.Post,
+                "query ($city: String!) { getCityByName(name: $city) { name country } }",
```

Comment:
By the way, `getCityByName` is bad naming for GraphQL data field.

Reply:
I only selected this endpoint and query because is an actual public API and does return data, even if a bit pointless.

Reply:
I understand.